sys/linux/init_alg.go

```diff
@@ -191,6 +195,8 @@ var allAlgs = map[int][]algDesc{
 		{"cbc(serpent)", nil},
 		{"cbc(twofish)", nil},
 		{"cbc-aes-aesni", nil},
+		{"cbc-aes-ce", nil},
+		{"cbc-aes-neon", nil},
```

Comment:
I am thinking maybe we could do some coarse-grained per-arch selection of algorithms. Annotating each of them with supported arches looks like lots of tedious work. But we just select, say, algorithms containing `-neon` only for `arm64`/`arm`, and `-sse` only for `amd64`/`386`, etc. This may give good bang for the buck. We have arch already available in `prog.Gen.Target.Arch`. But we will need to build separate pre-sorted lists for each arch.